pandas/tests/frame/methods/test_mode.py

```diff
@@ -0,0 +1,9 @@
+from pandas import DataFrame, Index
+import pandas._testing as tm
+
+
+def test_empty_df_mode():
+    df = DataFrame([], columns=["a", "b"])
+    result = df.mode()
+    expected = DataFrame([], columns=["a", "b"], index=Index([], dtype="int64"))
+    tm.assert_frame_equal(result, expected)
```

Comment:
Couldn't find any `DataFrame.mode` testing - guessing it relies on correctness from `Series.mode()` testing (in `pandas/tests/test_algos.py::TestMode`). However, empty result case doesn't hit `Series.mode`, so some `DataFrame` testing seems necessary. Is there a better place for this test?

Reply:
We keep them in `pandas/tests/frame/test_reductions.py`

Reply:
Ah thanks! Sorry for missing that